generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-mail/pom.xml

```diff
@@ -73,7 +73,7 @@
     <dependency>
       <groupId>com.icegreen</groupId>
       <artifactId>greenmail</artifactId>
-      <version>1.6.7</version>
+      <version>${quarkus-blaze-persistence.version}</version>
```

Comment:
Is this change intended? It looks like it's affecting a different artifact

Reply:
yes, that's a known issue. The generator is trying to figure out which property to use for versions in test POMs. Sometimes it maps them to an unrelated property that happens to have the needed value. It's weird but harmless. I think there are ways it can be improved. I should get to that at some point.

Reply:
@aloubyansky thanks for the explanation. That looks concerning because the intended version gets lost when `quarkus-blaze-persistence.version` is bumped to a new release.